### ISSUE TITLE
ffmpeg fails to encode some videos

### ISSUE THREAD

**Issue by dragondave** [learningequality/pressurecooker#11]
The file format we're encoding to requires the height in pixels to be divisible by two:

If we acquire a sample video:
`wget http://kennedyctr.vo.llnwd.net/o41/artsedge/videostories/culkin/culkin10.m4v`

And if we try to encode using the command from https://github.com/learningequality/pressurecooker/blob/master/pressurecooker/videos.py#L42
(including the `scale=....:-1`)
`ffmpeg -y -i culkin10.m4v -profile:v baseline -level 3.0 -b:a 32k -ac 1 -vf scale='480:-1' -crf 28 -preset slow -strict -2 -stats 'dragon.mp4'`

it fails:

> [libx264 @ 0x27e40a0] height not divisible by 2 (480x267)
> Error while opening encoder for output stream #0:0 - maybe incorrect parameters such as bit_rate, rate, width or height

But if we use -2, as per https://ffmpeg.org/ffmpeg-filters.html#scale

> If one and only one of the values is -n with n >= 1, the scale filter will use a value that maintains the aspect ratio of the input image, calculated from the other specified dimension. After that it will, however, make sure that the calculated dimension is divisible by n and adjust the value if necessary.

we get a successfully encoded video.

`ffmpeg -y -i culkin10.m4v -profile:v baseline -level 3.0 -b:a 32k -ac 1 -vf scale='480:-2' -crf 28 -preset slow -strict -2 -stats 'dragon.mp4'`

I'd also recommend we use `subprocess.check_call()`, catch the exception and maybe log an error: at the moment this just creates 0 byte files silently.

```
try:
    subprocess.check_output(["ffmpeg", "foo.mp4"], stderr=subprocess.STDOUT)
except Exception as e:
    print(e.output)
```

TL;DR: change the -1 to a -2 in `scale = "'{}:-1'"` on line 40 of pressurecooker's video.py

Tagging @jayoshih as they wrote the original code and might find this useful or know reasons not to do this.

**Comment by ivanistheone** [comment]
@dragondave  Thx for reporing this and finding the -2 option! This will be fixed in the next release.

In the meantime, use this workaround: pass `ffmpeg_settings={'crf':28}` when creating video files (i.e. do not set `max_width`).

This will use the "default" resolution setting video height to 480, and width as multiple of 2.
https://github.com/learningequality/pressurecooker/blob/master/pressurecooker/videos.py#L40

**Comment by ivanistheone** [comment]
fixed in https://github.com/learningequality/pressurecooker/pull/12
